NodeManager.cpp

```diff
@@ -1318,6 +1398,66 @@ void SensorBH1750::onReceive(const MyMessage & message) {
 #endif
 
 /*
+   SensorTSL2561
+*/
+#if MODULE_TSL2561 == 1
+// contructor
+SensorTSL2561::SensorTSL2561(int child_id): Sensor(child_id,A4) {
+  setPresentation(S_LIGHT_LEVEL);
+  setType(V_LEVEL);
+  _tsl = new TLS2561(TSL2561_ADDR_FLOAT);
+}
+
+// what do to during before
+void SensorTSL2561::onBefore() {
+  _tsl->begin();
+    // You can change the gain on the fly, to adapt to brighter/dimmer light situations
+  //tsl.setGain(TSL2561_GAIN_0X);         // set no gain (for bright situtations)
+  _tsl->setGain(TSL2561_GAIN_16X);      // set 16x gain (for dim situations)
```

Comment:
Would be nice to have setGain() and setTiming() exposed on SensorTSL2561 so the user can change the settings without modifying the core code